### DISCUSSION TITLE
Auto update m3u Playlists

### DISCUSSION THREAD

**Discussion by blastbeng** [navidrome/navidrome#3240] [Q&A] [answered]
Hello,
Does anyone knows if navidrome automatically updates manually made m3u playlists?

I have created a bunch of m3u files using a script that tries to match my music with Spotify recommendations.

If these m3u files changes does Navidrome modify the Playlist saved in its database by itself or do I need to delete the Playlist from the WebUI and let navidrome rescan my Playlist folder?

Thanks

**Comment by metalheim** [comment, marked as answer]
Navidrome should update the playlist if the m3u file changes.
Updates are only applied in one direction though (m3u - > navidrome), as navidrome never writes to your MusicFolder

**Reply by blastbeng** [reply]
Thanks, figured it out!
